design/open/pull-request.md

```diff
@@ -0,0 +1,28 @@
+# Pull-request Proposal
+
+Enable a way to submit changes to a branch by creating a request to merge changes.
+The pull-request captures the merge operation we request to apply.
+
+### Goals
+
+- Capture request to merge a branch or specific reference
+- Mechanism to review, discuss and approve changes before applying to a branch
+- Automation using actions can verify / provide feedback on pull-request
```

Comment:
I think we need to elaborate on it. `pull_request_open` trigger with actions that must pass?

Reply:
Added some description on the action - currently I think that an action 'pull-request' that will be triggered on creation and by request is enough. Until we support updates.